user-operator/src/main/java/io/strimzi/operator/user/operator/KafkaUserOperator.java

```diff
@@ -290,6 +290,8 @@ public KafkaUserOperator(Vertx vertx,
         return CompositeFuture.join(secretOperations.reconcile(namespace, KafkaUserModel.getSecretName(user), null),
                 aclOperations.reconcile(KafkaUserModel.getTlsUserName(user), null),
                 aclOperations.reconcile(KafkaUserModel.getScramUserName(user), null),
+                kafkaUserQuotasOperator.reconcile(KafkaUserModel.getTlsUserName(user), null),
+                kafkaUserQuotasOperator.reconcile(user, null),
```

Comment:
I did the test with a TLS user but in case for scram-sha why here it's not `KafkaUserModel.getScramUserName(user)`?

Reply:
According to https://github.com/strimzi/strimzi-kafka-operator/blob/a11e4b37b6d466467458062c63c4d2cecc4f5dcd/user-operator/src/main/java/io/strimzi/operator/user/model/KafkaUserModel.java#L343
it should be ok, or?

Reply:
that's true but not using the method makes the code less clear, as it's used above for deleting the ACLs